### ISSUE TITLE
Changing Servers

### ISSUE THREAD

**Issue by tpharaoh** [LaravelDaily/quickadmin#48]
This isn't really an "issue", rather a request for information.

When I went to put the code live, it of course runs migrations, and everything, and I love how easy to get it going.

However, I noticed at least the way I am moving it, everything re-creates ready to go, but, I need to manually add roles and export menus and menu_roles and import manually in new server. Am I just not doing this the smartest way?

**Comment by ModestasV** [comment]
Hello,
Yes, this is a thing that we were not planning at the beginning of the package - transfers to another server. At this stage our package is a generator that helps to generate project in not a very comfortable way - single generation. This means that when you generate everything you must manually transfer the base database to your new server and only then add new migrations to it. This is a problem and we saw it but at this point we are thinking of going big with this package.

In short: Export your generated SQL and Import it in new server. Only then run migrations if your migrations table is correct.

Sorry again and we hope that this won't cause much problems
